### PR TITLE
Fixes using teleporter with a GPS

### PR DESCRIPTION
**Pull request by monster860**
The `Get target from memory` button on the teleporter console is supposed to set the target to the GPS's saved location. Unfortunately, while the button's href is `lock`, the code references the href `locked`. I honestly have no idea why it worked before. From the history of it, this part of the code has not been changed in ages, (Before I started playing on this server!), I clearly remember having made use of this feature. Now, on the live server, it doesn't work. In other words, this is confusing as hell.

TL;DR: It's always been broken, but for some reason it used to work while it was broken, and stopped working.

### CODE REVIEW

**Comment by ProperPants** [comment]
It never worked at all for me. I'm eager to see that actually be useful now.
